test/benchmark_http_client_test.cc

```diff
@@ -26,6 +26,37 @@ using namespace testing;
 
 namespace Nighthawk {
 
+namespace {
+const std::shared_ptr<Envoy::Http::RequestHeaderMap>& getDefaultHeaderMap() {
+  static auto* header =
```

Comment:
While this is a known idiom when static storage on globals is required, I am trying to understand if it is necessary here.

Can we instead create the `std::shared_ptr<Envoy::Http::RequestHeaderMap>` instance as a member of the `BenchmarkClientHttpTest` test fixture? We could do this inside its constructor or inside the SetUp() method. Either of those would introduce simpler code for the posterity.

Reply:
Done.